src/get-started/wordcount-example.md

```diff
@@ -741,14 +742,14 @@ static class AddTimestampFn extends DoFn<String, String> {
 ```
 
 ```py
-This feature is not yet available in the Beam SDK for Python.
+# This feature is not yet available in the Beam SDK for Python.
 ```
 
 ### Windowing
 
-Beam uses a concept called **Windowing** to subdivide a `PCollection` according to the timestamps of its individual elements. PTransforms that aggregate multiple elements, process each `PCollection` as a succession of multiple, finite windows, even though the entire collection itself may be of infinite size (unbounded).
+Beam uses a concept called **Windowing** to subdivide a `PCollection` according to the timestamps of its individual elements. PTransforms that aggregate multiple elements process each `PCollection` as a succession of multiple, finite windows, even though the entire collection itself may be of infinite size (unbounded).
```

Comment:
"according to the timestamps" is true for all of the built-in window fns, but is not a requirement.

Reply:
changed to "... to subdivide a `PCollection` into a
bounded set of elements."